src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -608,10 +608,12 @@ export function getAppointmentSlots(startDate, endDate, forceFetch = false) {
           endDate: endDateString,
           useV2: featureVAOSServiceVAAppointments,
         });
-        const now = moment();
+        const tomorrow = moment()
```

Comment:
This is the main source of the bug. The calendar doesn't show today's slots, but we're not filtering them out here.